src/app/calculator-component/calculator-component.component.ts

```diff
@@ -1,15 +1,104 @@
 import { Component, OnInit } from '@angular/core';
 
+import { CalculatorService } from './../calculator.service';
+
 @Component({
   selector: 'app-calculator-component',
   templateUrl: './calculator-component.component.html',
-  styleUrls: ['./calculator-component.component.css']
+  styleUrls: ['./calculator-component.component.scss'],
+  providers: [CalculatorService]
 })
 export class CalculatorComponentComponent implements OnInit {
 
-  constructor() { }
+  state: string = "enter Number";
+  numbers: Array<number> = [];
+  operators: Array<string> = [];
+  result: Number = 0;
+  display: string = "";
+  operator: string = "Add";
+
+  constructor(private calcService: CalculatorService) { }
 
   ngOnInit() {
   }
 
+  enterNumber(number: string) {
+    // if the display already is showing the total result,
+    // and the user enter another digit again, 
+    // the display needs to be cleared to store the new number
```

Comment:
If you need a comment for this put into a function with a descriptive function name instead.